webpack.config.babel.js

```diff
@@ -0,0 +1,48 @@
+import webpack from 'webpack';
+import path from 'path';
+
+const { NODE_ENV } = process.env;
+const production = NODE_ENV === 'production';
+
+const filename = `redux-actions${production ? '.min' : ''}.js`;
+
+const plugins = [
+  new webpack.optimize.OccurenceOrderPlugin(),
+  new webpack.DefinePlugin({
+    'process.env.NODE_ENV': JSON.stringify(NODE_ENV)
+  })
+];
+
+if (production) {
```

Comment:
What do you think of `NODE_ENV === 'production'` inline here, since `production` is not used elsewhere and the triple-equals is already pretty readable?